src/frontend/screens/Observation/ObservationHeaderRight.tsx

```diff
@@ -24,8 +26,18 @@ export const ObservationHeaderRight = ({
     navigation.navigate('ObservationEdit', {observationId});
   }
 
-  const isMine = observationWithPreset.observation.createdBy === deviceId;
-  return isMine ? (
+  if (isLoading) {
+    return (
+      <UIActivityIndicator
+        size={20}
+        style={{alignItems: 'flex-end', marginRight: 20}}
+      />
+    );
+  }
+
+  const canEdit =
+    observationWithPreset.observation.createdBy === data?.deviceId || true;
```

Comment:
I got a message from @gmaclennan, He said to set the default value as true. Do you think I should change that? @gmaclennan

Reply:
For testing and building this feature, defaulting to true made sense. For production though we don't want people to be able to edit unless they own the observation. And since we cannot determine who owns the observation, lets not let anyone edit the observation

Reply:
changed